Snowdog_Components/components/Atoms/typography/heading/heading.config.js

```diff
@@ -146,5 +146,29 @@ module.exports = {
         text: 'Sixth level heading - tag and class'
       }
     },
+    {
+      name: 'page-class',
+      context: {
+        tag: 'h1',
+        class: 'heading heading--page',
+        text: 'Page level heading - only class'
+      }
+    },
+    {
+      name: 'page-tag-and-class',
```

Comment:
I find this variant unnecessary - the reason why level related classes have this kind of variant is because there are both:
- class selectors https://github.com/SnowdogApps/magento2-alpaca-theme/blob/7099fb5939d6e3ecce649f6d1cd817f31dd0b7d4/Snowdog_Components/components/Atoms/typography/heading/_heading.scss#L40
- tag selectors https://github.com/SnowdogApps/magento2-alpaca-theme/blob/7099fb5939d6e3ecce649f6d1cd817f31dd0b7d4/Snowdog_Components/components/Atoms/typography/heading/_heading.scss#L106

`heading--page` isn't related to any tag.